src/api/transaction/sign.js

```diff
@@ -48,12 +48,14 @@ function computeSignature(txJSON, keypair) {
 }
 
 function sign(txJSON: {Account: string; SigningPubKey: string,
-    TxnSignature: string}, secret: string):
+    TxnSignature: string}, secret: string, isRegular = false):
```

Comment:
Say your acvount id was account 30 from a seed, it loops until the keypair
matches the passed in address (hash of pubkey == address). Ill conceived.
Don't need to pass it.